### ISSUE TITLE
'linchpin init' doesn't setup workspace directory structure

### ISSUE THREAD

**Issue by yprokule** [CentOS-PaaS-SIG/linchpin#948]
Running `linchpin init` doesn't setup workspace directory
```bash
$ mkdir  new_linchpin_project
$ cd new_linchpin_project/
$ linchpin init 
Created destination workspace: /tmp/new_linchpin_project/dummy
$ tree 
.
├── dummy
│   ├── PinFile
│   ├── PinFile.json
│   └── README.rst
└── linchpin.log

1 directory, 4 files
$ linchpin --version
linchpin version 1.7.1
````

According to [docs](https://linchpin.readthedocs.io/en/latest/running_linchpin.html#initialization-init) it should create _inventories, layouts, topologies_ directories

**Comment by samvarankashyap** [comment]
@yprokule Thanks for pointing it out. Since recent version linchpin started following Single Pinfile structure where the layouts and topologies are included within the Pinfile and are optional. Moreover, The inventories, resources folders are created dynamically by linchpin. 
The documentation needs to be updated in accordance to the same.

**Comment by samvarankashyap** [comment]
fix on #951
